hs_access_control/management/commands/access_permissions.py

```diff
@@ -45,8 +45,8 @@ def handle(self, *args, **options):
         try:
             resource = get_resource_by_shortkey(resource_id, or_404=False)
         except BaseResource.DoesNotExist:
-            print("No such resource {}.".format(resource_id))
+            print(("No such resource {}.".format(resource_id)))
             usage()
             exit(1)
 
-        print(coarse_permissions(user, resource))
+        print((coarse_permissions(user, resource)))
```

Comment:
That was 2to3... not sure why it put those in.

Reply:
Looking into it, Looks like I could run 2to3 with a skip on print statements.  I think that should work fine.

https://docs.python.org/3.0/library/2to3.html#using-2to3
> When the -p is passed, 2to3 treats print as a function instead of a statement. This is useful when from __future__ import print_function is being used. If this option is not given, the print fixer will surround print calls in an extra set of parentheses because it cannot differentiate between the print statement with parentheses (such as print ("a" + "b" + "c")) and a true function call.